tests/expectations/tests/layout_array.rs

```diff
@@ -284,6 +284,13 @@ impl Default for rte_mempool_ops_table {
         unsafe { ::std::mem::zeroed() }
     }
 }
+impl ::std::cmp::PartialEq for rte_mempool_ops_table {
```

Comment:
I was unsure if this test output should be changed or not, but I think because all of the fields of the struct except the padding have PartialEq implementations, this should be valid (as long as the [rte_mempool_ops; 16] has PartialEq if rte_mempool_ops has PartialEq).

Reply:
Yeah, this looks ok, I think.